tests/integration/base.py

```diff
@@ -0,0 +1,299 @@
+import json
+import os
+import tempfile
+from abc import abstractmethod
+from collections import namedtuple
+from pathlib import Path
+from subprocess import Popen, PIPE, TimeoutExpired
+from logging import getLogger, StreamHandler, INFO
+from typing import Dict, Any, Optional, List
+from unittest import TestCase
+
+import pytest
+
+from tests import REPO_ROOT
+
+PROJECT_NAME = "project"
+CommandResult = namedtuple("CommandResult", "process stdout stderr")
+TIMEOUT = 300
+
+LOG = getLogger(__name__)
+LOG.addHandler(StreamHandler())
+LOG.setLevel(INFO)
+
+
+def run_command(command_list, cwd=None, env=None, timeout=TIMEOUT) -> CommandResult:
+    process_execute = Popen(command_list, cwd=cwd, env=env, stdout=PIPE, stderr=PIPE)
+    try:
+        stdout_data, stderr_data = process_execute.communicate(timeout=timeout)
+        LOG.info(f"Stdout: {stdout_data.decode('utf-8')}")
+        LOG.info(f"Stderr: {stderr_data.decode('utf-8')}")
+        return CommandResult(process_execute, stdout_data, stderr_data)
+    except TimeoutExpired:
+        LOG.error(f"Command: {command_list}, TIMED OUT")
+        LOG.error(f"Return Code: {process_execute.returncode}")
+        process_execute.kill()
+        raise
+
+
+class Base(object):
```

Comment:
Python 3 classes don't need to explicitly inherit `object`.

Reply:
👍 removed